0015-automatic-cluster-upgrades/0015-automatic-cluster-upgrades.md

```diff
@@ -0,0 +1,139 @@
+# RFC 0015 - Automatic Cluster Upgrades
+
+## User stories
+
+- As a user, and also as KaaS team provider, I would like to have the clusters running in the latest version automatically, and not worry about CVEs or EOL of the cluster components.
+
+- As a user, I would like to define maintenance windows to ensure clusters are not upgraded outside of these hours.
+
+- As a user I would like to mark a cluster as frozen and block upgrades for certain period of time.
+
+- As a user I want to define a policy that enforces some environments are upgraded before others to an agreed version.
+
+- As a user I want to schedule an upgrade on a certain day/time.
+
+- As a user, and also as KaaS team provider, I want to capture the result of upgrade.
```

Comment:
```suggestion
- As a customer, and also as Giant Swarm, we want to capture the result of an upgrade.
```

I'd like to see this ending in `, so that <whatever the goal is>` to explain why we should capture the result.